src/main/java/de/hysky/skyblocker/mixins/MushroomPlantBlockMixin.java

```diff
@@ -6,11 +6,15 @@
 import net.minecraft.block.*;
 import net.minecraft.util.shape.VoxelShape;
 import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.Unique;
 import org.spongepowered.asm.mixin.injection.At;
 
 @Mixin(MushroomPlantBlock.class)
-public class MushroomPlantBlockMixin extends Block {
+public abstract class MushroomPlantBlockMixin extends Block {
+	@Shadow
+	protected static VoxelShape SHAPE;
+
 	@Unique
 	private static final VoxelShape OLD_SHAPE = Block.createCuboidShape(4.8, 0.0, 4.8, 11.2, 6.4, 11.2);
```

Comment:
pretty sure this is now unused

Reply:
https://github.com/SkyblockerMod/Skyblocker/pull/1051/files#diff-2e45e569b555b94fdab5b3f4f7ea808eeaad80097053655aa8934da78f906c87R27

Reply:
oh right me dumb